mail_split_by_partner_conf/wizard/mail_compose_message.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api
+
+
+class MailComposer(models.TransientModel):
+    _inherit = 'mail.compose.message'
+
+    @api.model
+    def generate_email_for_composer(self, template_id, res_ids, fields=None):
+        # TODO - is need check case fields is none
```

Comment:
Could you check now? Or add a explicit message in the section Know Issues of the readme?

Reply:
I have done but forgot to remove it